stock_request/security/stock_request_security.xml

```diff
@@ -7,7 +7,7 @@
     </record>
     <record id="group_stock_request_user" model="res.groups">
         <field name="name">Stock Request User</field>
-        <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
+        <field name="implied_ids" eval="[(4, ref('stock.group_stock_user'))]" />
```

Comment:
Isn't one of the goals of this module to allow regular users to trigger stock needs?

Reply:
Sorry, changed